Example/Tests/TestLogger.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  TestLogger.swift
+//  LoggingKit_Tests
+//
+//  Created by Aleksandar Sergeev Petrov on 9.07.20.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import SFLoggingKit
+
+class TestLogger: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testLoggerShouldCallWritterLogMethod() {
```

Comment:
Correct me if I'm wrong, but I believe `Writter` in the method name is a typo of `Writer`. Same in the other related methods.